docs/developer-guide/contributing.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to zot Development
+
+> :point_right: The zot project is built for developers by developers. The zot project welcones the participation of the open source community in extending and improving zot. 
+
+## Submission Requirements
+
+> Summary: All contributions must meet these requirements:
+> 
+> - Adhere to the Apache license
+> - Be submitted by a pull request (PR) from your fork
+> - Commits must have a 
+> 
+### License
+
+zot is released under the [Apache License 2.0](https://github.com/project-zot/zot/blob/main/LICENSE). All contributions must adhere to this license.
+
+### Submitting a Pull Request (PR)
+
+First, fork the zot project on GitHub and submit a commit to your fork. Then open a new pull request (PR) to the zot project. All commits must meet these requirements:
+
+- Developer Certificate of Origin (DCO) and sign-off
+  
+  All commits require a Developer Certificate of Origin via the "Signed-off-by:" commit message and commit signatures using GPG keys. Include the `-s` flag in your commit command.
+
+- Commit message format
+
```

Comment:
The zot project follows the [Convention Commit Format](https://www.conventionalcommits.org)